packages/react/src/themes/teams/components/SplitButton/splitButtonStyles.ts

```diff
@@ -1,51 +1,90 @@
 import { ICSSInJSStyle } from '../../../types'
 import getBorderFocusStyles from '../../getBorderFocusStyles'
-import SplitButton from '../../../../components/SplitButton/SplitButton'
+import getIconFillOrOutlineStyles from '../../getIconFillOrOutlineStyles'
 
 const splitButtonStyles = {
-  button: ({ variables: v }): ICSSInJSStyle => ({
-    border: 0,
+  menuButton: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    borderTopRightRadius: 0,
+    borderBottomRightRadius: 0,
+    borderRightWidth: 0,
     padding: v.padding,
 
+    ...(p.small && {
+      height: v.smallDimension,
+      padding: v.smallPadding,
+      minWidth: v.smallMinWidth,
+      boxShadow: v.smallBoxShadow,
+    }),
+
     ':focus-visible': {
-      '::before': { border: 0 },
-      '::after': { border: 0 },
+      borderRightWidth: 0,
+
+      ':before': {
+        borderRightWidth: 0,
+      },
+
+      ':after': {
+        borderRightWidth: 0,
+      },
+    },
+
+    ':active': {
+      animationName: 'unset',
+      animationDuration: 'unset',
     },
-    ':active': { backgroundColor: v.backgroundColorActive },
   }),
+
+  toggleButton: ({ props: p, variables: v }): ICSSInJSStyle => ({
+    borderTopLeftRadius: 0,
+    borderBottomLeftRadius: 0,
+    borderColor: v.borderColor,
+    ...getIconFillOrOutlineStyles({ outline: true }),
+
+    ...(p.primary && {
+      borderWidth: v.borderWidthLeftOnly,
```

Comment:
Is `borderWidth` only needed for primary and disabled buttons?

Reply:
Kind of the opposite, only primary & disabled buttons only have the left side border, secondary buttons have borders on all sides. 

borders on all sides of the toggle:
![image](https://user-images.githubusercontent.com/6237759/70575043-6a246c80-1b5a-11ea-95ab-c00ad7f59f3c.png)

one border on the left of the toggle:
![image](https://user-images.githubusercontent.com/6237759/70575056-71e41100-1b5a-11ea-9279-004c7b2dafb0.png)